### PR TITLE
require_relative

### PR DESCRIPTION
**Pull request by xavierholt**
The current `gem` branch dies on me when trying to run `bin/twemproxy_exporter`:
```
[17:53:43 holt@Minerva twemproxy_exporter]$ bin/twemproxy_exporter 
/Users/holt/.rbenv/versions/2.4.0/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- twemproxy_exporter (LoadError)
	from /Users/holt/.rbenv/versions/2.4.0/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from bin/twemproxy_exporter:8:in `<main>'
```

This is the full change to using `require_relative`.  Should work all the way down to Ruby 1.9.2, which went EOL [a couple years ago](https://en.wikipedia.org/wiki/Ruby_(programming_language)#Table_of_versions); I assume breaking compatibility with 1.8 isn't a big problem.  Not sure the change for `spec_helper` was strictly necessary, but once the `gem` branch lands I'll add `value` getters that'll make the tests a lot cleaner and sidestep the helpers completely.

### CODE REVIEW

File: twemproxy_exporter.gemspec [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'twemproxy_exporter/version'
+require_relative 'lib/twemproxy_exporter/version'
```
Comment (atongen) [inline twemproxy_exporter.gemspec]: I'd rather not use `require_relative` here. When rubygems loads the gem, the `lib` directory should get added to the load path. This code was generated by bundler when the gem was built using `bundle gem twemproxy_exporter`.

It probably doesn't matter too much for this gem specifically, but in general I think it's cleaner and more idiomatic to add the lib dir to the load path and then use regular `require` statements.

**Review by atongen — CHANGES_REQUESTED** [review]
See comment about the gemspec.

File: twemproxy_exporter.gemspec (hunk 1 above)
Comment (xavierholt) [reply twemproxy_exporter.gemspec]: Fixed.  I didn't realize that rubygems itself was messing with the path.  Should I revert the executable as well?

Comment (atongen) [reply twemproxy_exporter.gemspec]: That's ok, I think this looks good. Thank you!

**Pull request merged**